jobs/workflow_cli_release.groovy

```diff
@@ -31,15 +31,15 @@ job("${repoName}-tag-release") {
   }
 
   parameters {
-    stringParam('RELEASE', defaults.workflow.release, 'Release to use for tag checkout')
-    booleanParam('RUN_E2E', false, 'check to run downstream release e2e job')
+    stringParam('TAG', '', 'Specific tag to release')
```

Comment:
@Joshua-Anderson good question as it's definitely not obvious from this change set.  The `TAG` env var is inspected in the previously-existing `get_latest_tag.sh` [script](https://github.com/deis/jenkins-jobs/blob/master/bash/scripts/get_latest_tag.sh), the use of which was added in this PR.